packages/jest-util/src/formatTestResults.js

```diff
@@ -12,29 +12,34 @@
 
 import type {
   AggregatedResult,
+  AssertionResult,
   CodeCoverageFormatter,
   CodeCoverageReporter,
+  FormattedTestAssertion,
+  FormattedTestResult,
+  FormattedTestResults,
   TestResult,
-  AssertionResult,
 } from 'types/TestResult';
 
 const formatResult = (
   testResult: TestResult,
   codeCoverageFormatter: CodeCoverageFormatter,
   reporter: CodeCoverageReporter,
-): Object => {
-  const output = ({
+): FormattedTestResult => {
+  const currTime = Date.now();
+  const output: FormattedTestResult = {
+    assertionResults: [],
+    coverage: {},
+    endTime: currTime,
     message: '',
     name: testResult.testFilePath,
+    startTime: currTime,
+    status: 'failed',
     summary: '',
-  }: any);
+  };
 
   if (testResult.testExecError) {
-    const currTime = Date.now();
-    output.status = 'failed';
-    output.message = testResult.testExecError;
```

Comment:
the original code did `output.message = testResult.testExecError` but on the right you access `.message`. Why?